src/components/ForgotPassword/ForgotPassword.css

```
@@ -0,0 +1,96 @@
+#password-recovery-form {
+	border: var(--border);
+	border-radius: var(--rounded-md);
+	display: flex;
+	flex-direction: column;
+	max-width: 400px;
```

suggest: let us use rem for consistency
```suggestion
	max-width: 25rem;
```